SimTracker/TrackTriggerAssociation/python/StubAssociator_cfi.py

```diff
@@ -0,0 +1,10 @@
+import FWCore.ParameterSet.Config as cms
+
```

Comment:
All cfi files should have a comment at start explaining what they are for. As we already have a TTStubAssociation_cfi.py, the comment here should make clear the difference between the two.